src/components/Card/useCardVariantBackground.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CardProps } from './Card';
+import { Theme } from '../../theme';
+import { BoxProps } from '../Box';
+
+type ThemeColor = keyof Theme['colors'];
+
+export const getBackgroundColor = (color: CardProps['variant']): ThemeColor => {
+  switch (color) {
+    case 'darker':
+      return 'navyblue-600';
+    case 'dark':
+      return 'navyblue-500';
+    case 'lighter':
+      return 'navyblue-300';
+    case 'light':
+    default:
+      return 'navyblue-400';
+  }
+};
+
+type useCardVariantBackgroundProps = Required<Pick<CardProps, 'variant'>>;
+
+const useCardVariantBackground = ({ variant }: useCardVariantBackgroundProps): BoxProps['bg'] => {
+  return React.useMemo(() => getBackgroundColor(variant), [variant]);
```

Comment:
Optional: no need for a `React.useMemo` here since you are only returning a string. We use `useMemo` to keep the samee reference on non-primitives (i.e. functions, objects & arrays).

Reply:
Good call! Thanks for the tip!